micromamba/tests/test_create.py

```diff
@@ -419,6 +419,49 @@ def test_channels(tmp_home, tmp_root_prefix, tmp_path, cli, yaml, env_var, rc_fi
         assert res["channels"] == ["conda-forge"]
 
 
+@pytest.mark.parametrize("shared_pkgs_dirs", [True], indirect=True)
+@pytest.mark.parametrize("env_vars", (False, True))
+@pytest.mark.parametrize("no_env", (False, True))
+def test_spec_file_env_vars(tmp_home, tmp_root_prefix, tmp_path, env_vars, no_env):
```

Comment:
I tested locally, and activating the environment successfully sets environment variables.

(I wonder whether there's a test checking that environment variables are set when the environment is activated.)